src/Writer/Pgsql.php

```diff
@@ -358,42 +346,47 @@ public function tablePrimaryKey($tableName)
         $stmt = $this->db->query(sprintf($query, $this->dbParams['schema'], $tableName));
         $res = $stmt->fetchAll(\PDO::FETCH_ASSOC);
 
-        return array_map(function($row) {
+        return array_map(function ($row) {
             return $row['column_name'];
         }, $res);
     }
 
-    public function getTableInfo($tableName)
+    public function getTableInfo(string $tableName): array
+    {
+        throw new ApplicationException('Method "getTableInfo" not implemented');
+    }
+
+
+    public function validateTable(array $tableConfig): void
     {
-        throw new ApplicationException("Method not implemented");
```

Comment:
treba tady muzem dat `return isTableValid($tableConfig);` i kdyz je to zatim vzdycky `true`

Reply:
hotovo :+1: